test/integration/detailed_export_test.rb

```diff
@@ -43,6 +43,10 @@ def test_to_export_with_watchers
   def test_to_export_with_journals
     assert_columns_options 'settings_journal'
   end
+  
+  def test_to_export_with_journals
```

Comment:
The method name is already used in this test class.

Reply:
Thank you, I'll fix!